webapp/src/components/App/Singlestudy/explore/Debug/Data/Text.tsx

```diff
@@ -111,24 +123,31 @@ function Text({ studyId, filePath, filename, canEdit }: DataCompProps) {
                 onUploadSuccessful={handleUploadSuccessful}
               />
             )}
-            <DownloadButton onClick={handleDownload} />
-          </Menubar>
-          <Box sx={{ overflow: "auto" }}>
-            <SyntaxHighlighter
-              style={atomOneDark}
-              lineNumberStyle={{
-                opacity: 0.5,
-                paddingRight: theme.spacing(3),
-              }}
-              customStyle={{
-                margin: 0,
-                padding: theme.spacing(2),
-                borderRadius: theme.shape.borderRadius,
-                fontSize: theme.typography.body2.fontSize,
-              }}
-              {...getSyntaxProps(text)}
+            <DownloadButton
+              onClick={handleDownload}
+              disabled={isEmptyContent(text)}
```

Comment:
For me it's not necessary to disable download for this

Reply:
yes, maybe the space are used I don't know, I wouldn't add a restriction for this

Reply:
you are right about this it should not be disabled mybad